### PR TITLE
[LER-23] - Program card

### PR DESCRIPTION
**Pull request by valenchus**
To easily test this PR:
1) you can download an image and put into the assets/images folder
2) then, replace the source from the image to: source={require('../../../../assets/images/your-image.jpeg')}
3) check the 3 different program cards by adding to a file:

      <StyledRow
        css={{ padding: 20, gap: 10, background: theme.primary900, width: 400, height: 200 }}
      >
        <ProgramCard id="uno" title="Programa 1" imgUrl="as" status="completed" progress={0.4} />
        <ProgramCard id="dos" title="Programa 2" imgUrl="as" status="in_progress" progress={0.1} />
        <ProgramCard id="tres" title="Programa 3" imgUrl="as" status="not_started" progress={0.2} />
      </StyledRow>

### CODE REVIEW

File: assets/icons/HeartPill.tsx [hunk 1]
```diff
@@ -0,0 +1,179 @@
+import { Defs, LinearGradient, Path, Rect, Stop, Svg } from 'react-native-svg';
+import { PillProps } from './types';
+
+const HeartPill = ({ status }: PillProps) => {
```
Comment (fabrizioserial) [inline assets/icons/HeartPill.tsx:4]: Eliminar este componente

File: assets/icons/types.ts [hunk 2]
```diff
@@ -2,3 +2,7 @@ export interface IconInterface {
   color?: string;
   size?: number;
 }
+
+export interface PillProps {
```
Comment (fabrizioserial) [inline assets/icons/types.ts:6]: porque aca esto?

File: src/components/program/ProgramCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  title: string;
+  imgUrl: string;
+  status: 'in_progress' | 'completed' | 'not_started';
+  progress?: number;
+}
+
+const ProgramCard = ({ id, title, imgUrl, status, progress }: ProgramCardProps) => {
+  const theme = useTheme();
+  return (
+    <>
```
Comment (fabrizioserial) [inline src/components/program/ProgramCard/index.tsx:19]: Este fragment no hace falta

File: src/components/program/ProgramCard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        key={id}
+        css={{
+          width: 109,
+          height: 145,
+          gap: 6,
+          alignItems: 'center',
+        }}
+      >
+        <StyledBox>
+          <Image style={{ width: 109, height: 109 }} source={{ uri: imgUrl }}></Image>
```
Comment (fabrizioserial) [inline src/components/program/ProgramCard/index.tsx:30]: Esto puede tener un Self-Closing  `<image ... />`

File: src/components/program/ProgramCard/index.tsx [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+      key={id}
+      css={{
+        width: 109,
+        height: 145,
+        gap: 6,
+        alignItems: 'center',
+      }}
+    >
+      <StyledBox>
+        <Image style={{ width: 109, height: 109 }} source={{ uri: imgUrl }} />
```
Comment (dipietronicolas) [inline src/components/program/ProgramCard/index.tsx:29]: borderRadius is missing

**Review by dipietronicolas — CHANGES_REQUESTED** [review]
..

**Review by dipietronicolas — APPROVED** [review]
LGTM

**Pull request merged**